myapp/src/components/InputField.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This is a cutome input field.
+ * This all params are required.
+ *
+ * @param {string} title - use for input labaling
+ * @param {string} value - user given current value
+ * @param {string} type - input tag type
+ * @param {Function} [ e => setfield(e.target.value)] onChange - call back function
+ * @param {string} error - display an error if the user given data invalid
+ * @returns {Component}  - return a input  component
```

Comment:
```suggestion
 * This is a custom input field.
 *
 * @param {string} title - input label
 * @param {string} value - current value
 * @param {string} type - input tag type
 * @param {Function} [ e => setfield(e.target.value)] onChange - callback function to fire when the input value changes
 * @param {string} error - error text to be displayed if the provided data is invalid
 * @returns {Component}  - return an input component
```
Some things to note here:
We should always correct spelling and grammatical errors we have with the text
The description of a function/param should be meaningful on it's own without having to go through the code to understand

Reply:
Added meaningful description.
aa439fed25ecd2380580da968e5ee9fca0a39698